.ssh/known_hosts

```diff
@@ -0,0 +1,6 @@
+github.com ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIOMqqnkVzrm0SdG6UOoqKLsabgH5C9okWi0dh2l9GKJl
```

Comment:
You better be glad I don't have a photographic memory, because I shouldn't have seen this. Add `.ssh` to `.gitignore`